webcommon/typescript.editor/external/binaries-list

```diff
@@ -15,22 +15,5 @@
 # specific language governing permissions and limitations
 # under the License.
 
-EEBA3C1183C93B924F59F99D18143C6854855140 command-exists-1.2.6.zip
-D54E3334BE790F2AF65068B404E069AB7158795B commander-2.20.3.zip
-9545FC9061706C762E852F3D4DA3164EDFD96FE1 crypto-random-string-1.0.0.zip
-97705D5EE06EDD1981369ABD129EAC37C25DC5D5 fs-extra-7.0.1.zip
-DB515913528F8F8C48678FEF88697875ADEF133C graceful-fs-4.2.6.zip
-5A908B2D235A118C634F58E228B3992488847A98 jsonfile-4.0.0.zip
-F176424C20EE59996C503BAEBF18D2C70F184840 p-debounce-1.0.0.zip
-C553EC2693BF9B1E942E8DDFFBF6A591FE753F84 temp-dir-1.0.0.zip
-5C31F32D7D10CAB54F13408C64B42B62D51796EE tempy-0.2.1.zip
-B36AEE158BE444FD89CA51DBE61EC2BA4E7653AA typescript-4.1.4.zip
-719BC3E8A6F6DDA67A7E9642B498386D6CAD022A typescript-language-server-0.5.1.zip
-80FD5AA5DF32BE742BE353E5C4784868ED7D0D38 unique-string-1.0.0.zip
-BE24C4558E26E8B7CDEAFF2F9AE70D3BB5BEB0A8 universalify-0.1.2.zip
-09CF27455281E6C4FAA30571125D4F6E42EC3D5D vscode-jsonrpc-6.0.0.zip
-06B648FCE1A5B1FBC0306C30DB593AAAF6989A52 vscode-languageserver-5.3.0-next.10.zip
-6065994D322B86EB06D642F96F16018774E8440F vscode-languageserver-protocol-3.16.0.zip
-BC9B7B5E8243882E6AFC2D26D1008EF3F960F22F vscode-languageserver-types-3.16.0.zip
-191490D1923C11D2D7BFDB8F86CEB61567CA62A0 vscode-textbuffer-1.0.0.zip
-75BF5DA9B1BFDFD1A85E0A67E86504B7E05482D8 vscode-uri-1.0.8.zip
+48304CC9611D4F3B4ACD5A6ABB88689913BFA7A0 https://doppel-helix.eu/test-typescript-update/48304CC9611D4F3B4ACD5A6ABB88689913BFA7A0-typescript-5.6.2.zip typescript-5.6.2.zip
+6895F0456E4B0FE3D857AA7D3F399932A026D060 https://doppel-helix.eu/test-typescript-update/typescript-language-server-4.3.3.zip typescript-language-server-4.3.3.zip
```

Comment:
@matthiasblaesing was this host intended? `test-typescrpt-update` sounds like a temporary location.

Reply:
@matthiasblaesing the files seem to be offline now :(

Reply:
Yes should have been done, I'm looking into this now.

Reply:
Update: https://github.com/apache/netbeans/pull/8168